java/src/jmri/Section.java

```diff
@@ -178,44 +188,51 @@ public void setState(int state) {
             firePropertyChange("state", old, mState);
             // update the forward/reverse blocking sensors as needed
             switch (state) {
-                case FORWARD:
+                case FORWARD: {
```

Comment:
What's the reason for the curly braces here? Curly braces are not needed in switch-case statements.

Reply:
They're also not necessary in single line if statements… but I prefer them anyway; it means you can double-click the brace (in most editors) and get everything up to the matching brace. Obviously without braces you can't do that.
For me they're right up there with conditional return statements: they're just a bug waiting to happen.